tracie/data_public/README.md

```diff
@@ -0,0 +1,27 @@
+This directory contains the training and test files for evaluating predictions,
+and a sample prediction file.
+
+The files `train_iid.jsonl` and `train_uniform.jsonl` are two versions of the
+training data that can be used. These two training sets refer to splits, based
+on a strict `iid` split and more balanced `uniform` split.
+
+The file `test.jsonl` has the test questions, without gold labels.
+
+Each example in these files looks like the following:
+
+```json
+{
+  "query": "event: Tom's teeth are crooked ends before he has braces on for a while",
+  "story": "Tom needed to get braces. He was afraid of them. The dentist assured him everything would be fine. Tom had them on for a while. Once removed he felt it was worth it.",
+  "gold_label": "contradiction"
+}
+```
+
+and consists of three fields:
+
+* `query` (or hypothesis)
+* `story` (or premise)
+* `gold_label` (the inference label; in `test.jsonl` this is hidden and replaced with `"-"`)
```

Comment:
Is this necessary? Can the field simply be skipped, and not given the value "-"?

Reply:
it can be skipped (it's consistent with a convention from some standard dataset).